app/components/App/App.scss

```diff
@@ -1,16 +1,16 @@
-@import 'styles/variables.scss';
+@import '../../../styles/variables.scss';
```

Comment:
This can use a relative path from the app dir:

`@import '../../../styles/variables.scss';` -> `@import 'styles/variables.scss';`